src/main/java/sirius/web/http/LangHelper.java

```diff
@@ -46,7 +46,7 @@ private LangHelper() {
             if (m.matches()) {
                 double q = Value.of(m.group(4)).asDouble(1.0d);
                 String language = m.group(1);
-                if (q > bestQ && NLS.isSupportedLanguage(language)) {
+                if (q > bestQ && UserContext.get().getUserManager().isSupportedLanguage(language)) {
```

Comment:
shoudln't we ask the ScopeInfo instead? add a method or transform into a proper interface(?) this has nothing to do with usermanagers(?)